.travis.yml

```
@@ -55,7 +55,7 @@ jobs:
         # Install and run `yamllint`
         - pip install --user yamllint
         # yamllint disable-line rule:line-length
-        - yamllint -s . .yamllint pillar.example test/integration/repositories/pillars.sls
+        - yamllint -s . .yamllint pillar.example test/salt/pillar/pillars.sls
```

```suggestion
        - yamllint -s . .yamllint pillar.example test/salt/pillar/pillars.sls test/salt/pillar/windows.sls
```

This would be the normal course of action.  However, this won't work at the current time with the use of Jinja in that file.  Will non-Windows OSes ever use it?